packages/rule-http-cache/src/rule.ts

```diff
@@ -98,7 +98,7 @@ export default class HttpCacheRule implements IRule {
              *
              * Live example: https://regex101.com/r/6VduJO/2/
```

Comment:
the `Live example` contains the old regex, can you update it too?

Reply:
nice catch